recipes/ncbi-cxx-toolkit-public/all/conanfile.py

```diff
@@ -0,0 +1,296 @@
+            if "" in _res:
+                _res.remove("")
+        return _res
+
+    def export(self):
+        copy(self, self._dependencies_filename,
+            os.path.join(self.recipe_folder, self._dependencies_folder),
+            os.path.join(self.export_folder, self._dependencies_folder))
+        copy(self, self._requirements_filename,
+            os.path.join(self.recipe_folder, self._dependencies_folder),
+            os.path.join(self.export_folder, self._dependencies_folder))
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        _alltargets = self._parse_option(self.options.with_targets)
+        _required_components = set()
+        for _t in _alltargets:
+            for _component in self._tk_dependencies["components"]:
+                _libraries = self._tk_dependencies["libraries"][_component]
+                if _t in _libraries:
+                    _required_components.add(_component)
+                    break
+
+        _allcomponents = self._parse_option(self.options.with_components)
+        _required_components.update(_allcomponents)
+
+        if len(_required_components) > 0:
+            _todo = _required_components.copy()
+            _required_components.clear()
+            _next = set()
+            while len(_todo) > 0:
+                for _component in _todo:
+                    if not _component in _required_components:
+                        _required_components.add(_component)
+                        if _component in self._tk_dependencies["dependencies"].keys():
+                            for _n in self._tk_dependencies["dependencies"][_component]:
+                                if not _n in _required_components:
+                                    _next.add(_n)
+                _todo = _next.copy()
+                _next.clear()
+
+        if len(_required_components) == 0:
+            _required_components.update( self._tk_dependencies["components"])
+        else:
+            for component in _required_components:
+                self._componenttargets.update(self._tk_dependencies["libraries"][component])
+
+        requirements = set()
+        for component in _required_components:
+            libraries = self._tk_dependencies["libraries"][component]
+            for lib in libraries:
+                if lib in self._tk_dependencies["requirements"].keys():
+                    requirements.update(self._tk_dependencies["requirements"][lib])
+
+        for req in requirements:
+            pkgs = self._translate_req(req)
+            if pkgs is not None:
+                for pkg in pkgs:
+                    print("Package requires ", pkg)
```

Comment:
```suggestion
```

Please, do not use `print()`, prefer `self.output.info()`. Requirements you can avoid, because is already listed by conan when solving the graph.